Bottom: 3f6774c0f24d136d84ced9c2ac645d6fda4eb712
Top:    3c41fbbc810dfaa818b35f2b33d56afac55da2cb
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-04 08:06:31 -0400

Unbreak the mpc5200 i2s Kconfig

Unbreak the mpc5200 i2s Kconfig

Signed-off-by: Jon Smirl <jonsmirl@gmail.com>


---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index c7339f3..dc79bdf 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -24,7 +24,7 @@ config SND_SOC_MPC8610_HPCD
 
 config SND_SOC_MPC5200_I2S
 	tristate "Freescale MPC5200 PSC in I2S mode driver"
-	depends on PPC_MPC52xx && PPC_BESTCOMM && BROKEN
+	depends on PPC_MPC52xx && PPC_BESTCOMM
 	select SND_SOC_OF_SIMPLE
 	select SND_MPC52xx_DMA
 	select PPC_BESTCOMM_GEN_BD
